.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ on:
 
 jobs:
   python:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
Staying on 22 for now to avoid build errors: https://github.com/foxglove/foxglove-python/actions/runs/12356809802/job/34535840957. Runners have been rolling out 24.04 as "latest" this month.